src/window.rs

```diff
@@ -1398,15 +1399,14 @@ impl Window {
     ///
     /// ```no_run
     /// # use winit::dpi::{LogicalPosition, PhysicalPosition};
-    /// # use winit::event_loop::EventLoop;
     /// # use winit::window::Window;
-    /// # let mut event_loop = EventLoop::new().unwrap();
-    /// # let window = Window::new(&event_loop).unwrap();
+    /// # fn scope(window: &Window) {
```

Comment:
Maybe?
```suggestion
    /// # let window: &Window = unimplemented!();
```
Since the examples are `no_run` anyways.

Reply:
I'm not sure about it since all the code in examples will be kind of _dead_ with such pattern.

Reply:
Yeah, on the other hand, if someone forgets `no_run`, then the example will still mistakenly run, it'll just do nothing.

Reply:
I'd still probably go with `fn` for now.